build failure under cygwin

My most recent successful build was 0.5.0-dev+4506 at cd2a278.
```
HAW7L0605$ make
    LINK usr/bin/libjulia.dll
/usr/lib/gcc/x86_64-w64-mingw32/4.9.2/../../../../x86_64-w64-mingw32/bin/ld: cannot find -lLLVM-3.7
collect2: error: ld returned 1 exit status
Makefile:219: recipe for target '/home/s2sajs/julia-master/usr/bin/libjulia.dll' failed
make[1]: *** [/home/s2sajs/julia-master/usr/bin/libjulia.dll] Error 1
Makefile:90: recipe for target 'julia-src-release' failed
make: *** [julia-src-release] Error 2
```

--- a/deps/llvm.mk
+++ b/deps/llvm.mk
@@ -87,7 +87,7 @@ LLVM_FLAGS += --disable-profiling --enable-static $(LLVM_TARGET_FLAGS)
 LLVM_FLAGS += --disable-bindings --disable-docs --disable-libedit --disable-terminfo
 # LLVM has weird install prefixes (see llvm-$(LLVM_VER)/build_$(LLVM_BUILDTYPE)/Makefile.config for the full list)
 # We map them here to the "normal" ones, which means just prefixing "PROJ_" to the variable name.
-LLVM_MFLAGS := PROJ_libdir=$(build_libdir) PROJ_bindir=$(build_depsbindir) PROJ_includedir=$(build_includedir)
+LLVM_MFLAGS := PROJ_libdir=$(build_libdir) PROJ_bindir=$(build_depsbindir) PROJ_includedir=$(build_includedir) DestSharedLibDir=$(build_shlibdir)
 ifeq ($(LLVM_ASSERTIONS), 1)
 LLVM_FLAGS += --enable-assertions
 LLVM_CMAKE += -DLLVM_ENABLE_ASSERTIONS:BOOL=ON

